core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/scatter3DChart/Scatter3dChartOpDesc.scala

```diff
@@ -0,0 +1,121 @@
+package edu.uci.ics.texera.workflow.operators.visualization.scatter3DChart
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.{JsonSchemaInject, JsonSchemaTitle}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType, Schema}
+import edu.uci.ics.texera.workflow.operators.visualization.{
+  VisualizationConstants,
+  VisualizationOperator
+}
+
+@JsonSchemaInject(json = """
+{
+  "attributeTypeRules": {
+    "title": "string"
+  }
+}
+""")
+class Scatter3dChartOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+  @JsonProperty(value = "title", required = true)
+  @JsonSchemaTitle("Plot Title")
+  @JsonPropertyDescription("The value for the plot tile")
```

Comment:
title?

Reply:
I think I don't get what the value = "title" mean. Could you explain this field to me? What time do I need to use the value variable?

Reply:
I only mean you have a typo in line 25, `@JsonPropertyDescription("The value for the plot tile")`. It should be `title` instead of `tile`.

Reply:
OK, got it. My bad, I have fixed it. Thank you.